CMakeLists.txt

```diff
@@ -231,6 +231,7 @@ if (MFEM_USE_HIP)
     list(INSERT CMAKE_PREFIX_PATH 0 ${ROCM_PATH})
   endif()
   find_package(HIP REQUIRED)
+  find_package(HIPBLAS REQUIRED)
```

Comment:
We should add `cuBLAS` dependency too, I suspect. In a parallel build, HYPRE may add that but for serial build we'll need it.

Reply:
Is this the right way to do it? I'm not very familiar with how to set things up with our CMake build system (as you may be able to tell…)

https://github.com/mfem/mfem/blob/bc729465d6117a4b9a1bf5235ee4239f3d021c1c/CMakeLists.txt#L150

Reply:
This is one place -- looks good to me. Search for `CUSPARSE` for other places, if you have not done that already.